101-custom-rp-with-function/azuredeploy.json

```diff
@@ -4,18 +4,12 @@
     "parameters": {
         "location": {
             "type": "string",
+            "defaultValue": "eastus",
             "allowedValues": [ "australiaeast", "eastus", "westeurope" ],
             "metadata": {
                 "description": "Location for the resources."
             }
         },
-        "deployFunction": {
```

Comment:
why are we removing the condition?  It was put the explicitly to show it wasn't needed every time...